backend/.gitignore

```diff
@@ -1,4 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+.env
 
 # dependencies
 /node_modules
```

Comment:
Hi Kevin, are you writing a whole new gitignore instead of using the gitignore that I already wrote?

Reply:
As well, the .env file in the backend folder is already added in the root .gitignore file on line 439...

Reply:
I wanted to cover the case where ide projects were inside this directory. 

Like for instance, my ide has all its files in ./idea folder. While /.idea is in the original gitignore, /backend/.idea is not. This file covers that, such that git will ignore all ide files in this folder. 

Alternatively, I could have copy/pasted your original gitignore, and added in /backend/.idea, /backent/.vscode, to every single relative path in the file, but I did not want to spend my precious time doing that. I would recommend not bothering to change this, since it wont benefit us in any meaningful way. Its "good enough".